proposals/stage-0/stage-1/community-board-representation/README.md

```diff
@@ -0,0 +1,64 @@
+# Community Board Representation
+>  Stage 1
+
+## Champion
+
+[Myles Borins](https://github.com/MylesBorins)
+
+## Description
+
+### What exists today
+
+Currently the Node.js foundation has up to 3 community board seats:
+
+* 1 board seat for TSC
+* 1 board seat for Community Committee
+* 1 board seat for Individual Membership Program
+
+There are some limitations on these seats as noted in [Section 4.3](https://github.com/nodejs/board/blob/master/by-laws.md#section-43-nomination-election-and-term-of-office-of-directors) of the bylaws,
+specifically that:
+
+4.3 d) There must be 3 platinum directors for the individual membership to elect a board member
+
+4.3 e) There must be 2 platinum directors for the TSC or CommComm to elect a board member
+
+### Proposal for $FOUNDATION
+
+There would be up to 3 community board seats:
+
+* 1 board seat for CPC
+* 1 floating board seat to be delegated by CPC
+  - delegated on an annual basis and approved by Board
+* 1 board seat for the individual membership program
```

Comment:
It's hard to +1 this without understanding what the ind. membership program will look like, what kind of benefits we provide individual members vs. corp. members, how they can join, etc. For the record I am +1 to the idea, but I am also aware that this arrangement can create tension if it's not designed correctly (everyone has to have "skin in the game" and everyone needs to understand what it means to serve as the individual director).